--- a/gplaycli/gplaycli.py
+++ b/gplaycli/gplaycli.py
@@ -249,9 +249,9 @@ class GPlaycli:
 
 			if filename is None:
 				if self.append_version:
-					filename = detail['docId']+ "-v." + detail['versionString'] + ".apk"
+					filename = detail['docid']+ "-v." + detail['versionString'] + ".apk"
 				else:
-					filename = detail['docId']+ ".apk"
+					filename = detail['docid']+ ".apk"
 
 			logger.info("%s / %s %s", position, len(pkg_todownload), packagename)
 
@@ -300,7 +300,7 @@ class GPlaycli:
 						for obb_file in additional_data:
 							obb_filename = "%s.%s.%s.obb" % (obb_file["type"],
 															 obb_file["versionCode"],
-															 data_iter["docId"])
+															 data_iter["docid"])
 							obb_filename = os.path.join(download_folder, obb_filename)
 							obb_total_size = int(obb_file['file']['total_size'])
 							obb_chunk_size = int(obb_file['file']['chunk_size'])
@@ -371,7 +371,7 @@ class GPlaycli:
 					  if result['installationSize'] > 0 else 'N/A',
 					  result['numDownloads'],
 					  result['uploadDate'],
-					  result['docId'],
+					  result['docid'],
 					  result['versionCode'],
 					  "%.2f" % result["aggregateRating"]["starRating"]
 					  ]
-- 
2.25.0

